src/Gui/propertyeditor/PropertyItem.cpp

```diff
@@ -24,14 +24,18 @@
 #include "PreCompiled.h"
 
 #ifndef _PreComp_
-# include <QApplication>
+# include <algorithm>
 # include <QComboBox>
 # include <QFontDatabase>
 # include <QLocale>
 # include <QPalette>
 # include <QPixmap>
 # include <QTextStream>
 # include <QTimer>
+# include <QApplication>
+# include <QPalette>

```

Comment:
This is double-included -- @donovaly can you take a look at the includes here and see if it's just confused by the recent changes you've been making to the includes?

Reply:
`# include <QPalette>`

Is already in the file, so this should not be added again.

`# include <QApplication>`

should only be included if necessary.

besides this, please add includes sorted. This way doubled entries can be avoided.

Reply:
Thanks, this is now resolved.